convert.py

```diff
@@ -104,6 +115,28 @@ def GenerateAudioContentForText(text):
             break
     
 
+def eupub_to_chapters():
+    path = os.path.abspath(EPUB_FILEPATH)
+    print(path)
+    book = epub.read_epub(path)
+
+    full_text = ''
+    for text in book.get_items_of_type(ebooklib.ITEM_DOCUMENT):
+        html_content = text.get_content().decode("utf-8")
+        full_text += html_content
+        
+    chapter_text = convert_utils.get_all_chapters(full_text) 
+    print("{} Chapters found in ebook".format(len(chapter_text))) # FIXME
+
+
+    # For debugging only: Write the text contents to file
+    if OUTPUT_EPUB_TEXT_TO_TEST_FILE:
+        f = open("chapter_test.txt", "w")
+        for i, chapter in enumerate(chapter_text):
+            f.write("!!Chapter {}:\n{}".format(i+1,chapter))

```

Comment:
A quick comment would be appropriate here to explain this formatting :)

Reply:
Ack - #allen's weird debug strings